src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -468,7 +468,7 @@ const EnvironmentPicker = ({
         resources: Array<IAzureResource | string>
     ) => {
         dispatch({
-            type: EnvironmentPickerActionType.HANDLE_CONTAINER_CHANGE,
```

Comment:
Have not been able to validate the header in the container list shows the storage account instead of subscription since that dropdown doesn't seem to populate with remote values now. It only shows the single manual entry.
![image](https://user-images.githubusercontent.com/57726991/186236110-81829ff1-2387-45f3-a772-0b2850ed6bf9.png)

Reply:
I am not going to change it now as I still think all the resource pickers should be consistent with headers to show the resources divided by subscription. We can change it later if this is what customers want to see.

Reply:
subscription doesn't tell them anything in this case. It's better to hide the header so that it's not more confusing. This would imply that they might see data from another subscription in some case but that won't happen.